### PR TITLE
Develop Commit

### PR DESCRIPTION
**Pull request by JCeluch**
Add:
– grutnfile.js
– images folder
– index.html
– package.json
– sass folder

### CODE REVIEW

File: gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+module.exports = function(grunt) {
+
+  // Project configuration.
+  grunt.initConfig({
+  	sass: {
+      options: {
```
Comment (matwjablonski) [inline gruntfile.js]: create proper alignment :)

Comment (matwjablonski) [inline gruntfile.js]: we should see at first look what is in what object

File: gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            spawn: false,
+        },
+      } 
+    }
+  });
+  // Load the plugins tasks 
+  	grunt.loadNpmTasks('grunt-sass');
+    grunt.loadNpmTasks('grunt-contrib-imagemin');
+    grunt.loadNpmTasks('grunt-contrib-watch');
+  // Default task(s).
```
Comment (matwjablonski) [inline gruntfile.js]: where is BrowserSync? :)

File: gruntfile.js (hunk 1 above)
Comment (JCeluch) [inline gruntfile.js]: > we should see at first look what is in what object

What do you mean? I don't understand :)

**Pull request merged**